make/common/JavaCompilation.gmk

```diff
@@ -264,7 +264,7 @@ define SetupJavaCompilationBody
   endif
 
   # Allow overriding on the command line
-  JAVA_WARNINGS_ARE_ERRORS ?= -Werror
+  # JAVA_WARNINGS_ARE_ERRORS ?= -Werror
```

Comment:
This is not acceptable: we don't want to ignore warnings. If the warnings are in OpenJCEPlus code, those should be addressed, or failing that, `openjceplus` could be added to the list of `WARNING_MODULES` in `custom-spec.gmk.in`.